Tensile/Source/lib/include/Tensile/ContractionProblemPredicates.hpp

```diff
@@ -1055,6 +1055,41 @@ namespace Tensile
                     return problem.stridedBatched() == value;
                 }
             };
+
+            struct CUEfficiency : public Predicate_CRTP<CUEfficiency, ContractionProblem>
+            {
+                enum
+                {
+                    HasIndex = false,
+                    HasValue = false
+                };
+                bool value;
+
```

Comment:
HasValue = false, we can remove the value :)

Reply:
Haha good catch